pkg/kubectl/cmd/taint.go

```diff
@@ -40,6 +40,13 @@ import (
 	utiltaints "k8s.io/kubernetes/pkg/util/taints"
 )
 
+const (
+	MODIFIED    = "modified"
```

Comment:
Wherever we decide to put it, I think we should just use standard verbs: Added, Removed, Modified.